### DISCUSSION TITLE
Rework payment-provider-klarna

### DISCUSSION THREAD

**Discussion by jonathanelmgren** [pimcore/pimcore#12317] [Ideas]
Hello!

I was about to integrate Klarna payment provider to my ecommerce-shop but it looks like it is based on KCO v2 which is deprecated. An idea would be to rework the bundle to KCO v3 (https://docs.klarna.com/klarna-checkout/api/checkout-api/). 

I am happy to help in any way I can, I have worked with the klarna API before. However I am very limited in my PHP, let alone Pimcore knowledge at this point :)

**Comment by kingjia90** [comment]
That would be very appreciated, if you need help you can always write there or in the bundle repository
